rpc/src/client.rs

```diff
@@ -1,31 +1,45 @@
 //! Tendermint RPC client
 
-use bytes::buf::ext::BufExt;
-use hyper::header;
+use tendermint::{
+    abci::{self, Transaction},
+    block::Height,
+    evidence::Evidence,
+    net, Genesis,
+};
 
-use tendermint::abci::{self, Transaction};
-use tendermint::block::Height;
-use tendermint::evidence::Evidence;
-use tendermint::net;
-use tendermint::Genesis;
-
-use crate::{endpoint::*, Error, Request, Response};
+use crate::{
+    client::{
+        subscription::SubscriptionManager,
+        transport::{http_ws::HttpWsTransport, Transport},
```

Comment:
Still quite suprised to find these nested imports, they make it hard to edit them and don't help with readability either. I'd like to stick to the import style the crate has followed so far.

Reply:
Sure, I'll change them. My IDE was doing weird things, mixing the two styles, so I just went with this style. I'll reformat them 👍